nas/dispatch.go

```diff
@@ -19,6 +19,10 @@ func Dispatch(ue *context.AmfUe, accessType models.AccessType, procedureCode int
 		return errors.New("GSM Message should include in GMM Message")
 	}
 
+	if ue.State[accessType] == nil {
+		return errors.New("UE State is empty. Can't send GSM Message")
```

Comment:
Please output accessType to errors as below:

`return fmt.Errorf("UE State is empty (accessType=%q). Can't send GSM Message", accessType)`

Reply:
Added accessType to errors as suggested.